cgi-bin/LJ/Talk.pm

```diff
@@ -1830,7 +1831,8 @@ sub talkform {
 
             $ret .= "<input type='hidden' name='cookieuser' value='$remote->{'user'}' id='cookieuser' />\n";
             if ($screening eq 'A' ||
-                ($screening eq 'F' && !$journalu->trusts_or_has_member( $remote ))) {
+                ($screening eq 'F' && !$journalu->trusts_or_has_member( $remote )) ||
+                ( $journalu->has_autoscreen( $remote ) )) {
```

Comment:
Same as above.  I also see a similar construction at L1662 and L1775; do those also need updating?  They are used for OpenID users, but presumably those can be subject to selective screening as well.